src/formats/LAMMPSAtom.cpp

```diff
@@ -354,6 +354,8 @@ void LAMMPSAtomFormat::read_next(Frame& frame) {
     }
     std::vector<lammps_atom_attr_t> fields;
     fields.reserve(atoms_item.size() - 1);
+    std::vector<std::string> field_names;
+    field_names.reserve(atoms_item.size() - 1);
```

Comment:
If you feel like it, these two vectors could be made into a single one by creating an `AtomField` struct

```cpp
struct AtomField {
    std::string name;
    lammps_atom_attr_t kind;
};

// ...

std::vector<AtomField> fields;

// ...
```

Reply:
Done.